go/tendermint/apps/staking/state.go

```diff
@@ -35,6 +37,12 @@ var (
 	//
 	// Value is a little endian encoding of an uint64.
 	debondingIntervalKeyFmt = keyformat.New(0x54)
+	// acceptableTransferPeersKeyFmt is the key format used for the acceptable transfer peers set.
+	//
+	// Value is a CBOR-serialized map from acceptable runtime IDs the boolean true.
+	acceptableTransferPeersKeyFmt = keyformat.New(0x55)
```

Comment:
I assume the size of this map is expected to be small (on the order of number of runtimes)? Another option would be to have the entries be stored in the tree -- e.g., one key per entry:
```golang
keyformat.New(0x55, &signature.MapKey{})
```

Reply:
Right, that's the expectation. Thanks for posting another option. Let's keep that in mind so we can later support a large number of acceptable runtimes if we want to.